source/Mlos.Python/mlos/Grpc/OptimizerMicroservice.py

```diff
@@ -133,8 +133,10 @@ def Suggest(self, request, context): # pylint: disable=unused-argument
 
         # TODO: return an error if optimizer not found
         #
+        if request.Context.ContextJsonString != "":
+            raise NotImplementedError("context not supported in remote optimizers")
         with self.exclusive_optimizer(optimizer_id=request.OptimizerHandle.Id) as optimizer:
-            suggested_params = optimizer.suggest(random=request.Random, context=request.Context)

```

Comment:
Should we add a `# TODO` comment on adding support for handling `Context` in the future.

Reply:
Done. I think my thought was that the errors are implicit TODOs, we clearly want to support that soon-ish.